packages/lesswrong/components/tagging/WikiGradeDisplay.tsx

```diff
@@ -22,7 +23,7 @@ const wikiGradeDescriptions = {
 
 const WikiGradeDisplay = ({wikiGrade, classes}: {wikiGrade:number, classes: any}) => {
   const { LWTooltip } = Components
-  if (wikiGrade === 0) return null
+  if (forumTypeSetting.get() === 'EAForum' || wikiGrade === 0) return null
```

Comment:
Seems OK, but maybe at the wrong level. Feels like you shouldn't be loading the `WikiGradeDisplay` component at all in the parent. Maybe this is cleaner though.

Reply:
What I like about this is that it means that if it gets used somewhere else it doesn't matter. If I controlled all the code I would prefer not to load it, but I think in a collaboration, this makes sense.